src/redux/wallet/actions.js

```diff
@@ -151,21 +153,15 @@ export const withdrawTIME = (amount) => async (dispatch) => {
 //   })
 // }
 
+const paginator = new BlockDataPaginator(new WalletTxsProvider())
+paginator.provider.account = LS.getAccount()
+paginator.sizePage = 2 // TODO 20
```

Comment:
Moreover, we have approved size 10 for absolutely all pages, so just put 10 by default into the BlockDataPaginator